test/eslint

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bats
+set -eu
+
+load .env
+
+@test "$(basename "$BATS_TEST_FILENAME" | cut -d. -f1) --version" {
+  expected="$VERSION_REGEX"
+  run run_docker_binary "$BATS_IMAGE" "$(basename "$BATS_TEST_FILENAME" | cut -d. -f1)" --version
```

Comment:
Would be a bit more readable if `$(basename "$BATS_TEST_FILENAME" | cut -d. -f1)` is assigned to a variable

Reply:
Indeed. Fixed 👍🏻